bulk/eng_bulk_load-AssetRecord.csv

```diff
@@ -122,465 +128,481 @@ CGCON-CWFPCP-10302,A00847.1,Sensor,0,Coastal Wire-Following Profiler Controller,
-CGCON-GLDRCP-00336,OL000348,Sensor,1,Mobile Asset Controller for Coastal Glider,GL336 Controller,Teledyne Webb,G2 Slocum Gliders,336,,,,,
-CGCON-GLDRCP-00339,OL000349,Sensor,1,Mobile Asset Controller for Coastal Glider,GL339 Controller,Teledyne Webb,G2 Slocum Gliders,339,,,,,
-CGCON-GLDRCP-00340,OL000350,Sensor,1,Mobile Asset Controller for Coastal Glider,GL340 Controller,Teledyne Webb,G2 Slocum Gliders,340,,,,,
-CGCON-GLDRCP-00374,OL000351,Sensor,1,Mobile Asset Controller for Coastal Glider,GL374 Controller,Teledyne Webb,G2 Slocum Gliders,374,,,,,
-CGCON-GLDRCP-00375,OL000352,Sensor,1,Mobile Asset Controller for Coastal Glider,GL375 Controller,Teledyne Webb,G2 Slocum Gliders,375,,,,,
-CGCON-GLDRCP-00376,OL000353,Sensor,1,Mobile Asset Controller for Coastal Glider,GL376 Controller,Teledyne Webb,G2 Slocum Gliders,376,,,,,
-CGCON-GLDRCP-00379,OL000354,Sensor,1,Mobile Asset Controller for Coastal Glider,GL379 Controller,Teledyne Webb,G2 Slocum Gliders,379,,,,,
-CGCON-GLDRCP-00380,OL000355,Sensor,1,Mobile Asset Controller for Coastal Glider,GL380 Controller,Teledyne Webb,G2 Slocum Gliders,380,,,,,
-CGCON-GLDRCP-00387,OL000356,Sensor,1,Mobile Asset Controller for Coastal Glider,GL387 Controller,Teledyne Webb,G2 Slocum Gliders,387,,,,,
-CGCON-GLDRCP-00388,OL000357,Sensor,1,Mobile Asset Controller for Coastal Glider,GL388 Controller,Teledyne Webb,G2 Slocum Gliders,388,,,,,
-CGCON-GLDRCP-00389,OL000358,Sensor,1,Mobile Asset Controller for Coastal Glider,GL389 Controller,Teledyne Webb,G2 Slocum Gliders,389,,,,,
-CGCON-GLDRCP-00564,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL564 Controller,Teledyne Webb,G2 Slocum Gliders,564,,,,,
-CGCON-GLDRGP-00470,,Sensor,1,Mobile Asset Controller for Profiling Glider,,Teledyne Webb,G2 Slocum Gliders,470,,,,,Woods Hole Oceanographic Institution
+CGCON-CWFPCP-99105,,Sensor,0,Coastal Wire-Following Profiler Controller,WFP COASTAL Electronics #1,McLane,MMP-7,ML12991-05,,20130510,0,Early on, this WFP had controller brains swapped with 12991-03 (belonging to EA),Woods Hole Oceanographic Institution
+CGCON-ECPM01-00002,OL000201,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,2,,,,"CE06ISSM-00001-CPM1 CE01ISSM-0000(1,2,4)-CPM1",Oregon State University
+CGCON-ECPM01-00015,OL000208,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,15,,,,CE06ISSM-00004 CE06ISSM-00002-CPM1,Oregon State University
+CGCON-ECPM01-00016,OL000295,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,3605-20018-0016,,,,"CE01ISSM-0000(3,5)-CPM1",Oregon State University
+CGCON-ECPM01-00020,,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,20,,,,CE01ISSM-00006-SBC11,Oregon State University
+CGCON-ECPM01-01701,,Sensor,0,Surface Buoy Communications and Power Manager,Communications and Power Manager,WHOI,CPM,1701,,,,,
```

Comment:
According to eng_bulk_load-AssetRecord_NEW_2019-05-17 spreadsheet this entry should have Oregon State University added for MIO